headers/functions/overlay29.h

```diff
@@ -18,10 +18,13 @@ bool FixedRoomIsSubstituteRoom(void);
 bool StoryRestrictionsEnabled(void);
 int GetScenarioBalanceVeneer(void);
 void FadeToBlack(void);
+struct trap* GetTrapData(struct entity* trap_entity);
```

Comment:
Do `GetTrapData` and `GetItemData` just return the `info` field? Can we call these `GetTrapInfo` and `GetItemInfo` if so?

Reply:
Sure